lms/static/scripts/frontend_apps/components/JSTORPicker.js

```diff
@@ -219,6 +227,39 @@ export default function JSTORPicker({ onCancel, onSelectURL }) {
             </div>
           )}
 
+          {metadata.data && (
+            <>
+              <div className="grow" />
+              <div className="flex space-x-2 items-center px-1">
+                <label htmlFor="accept-jstor-terms" className="grow text-right">
```

Comment:
We have a `useUniqueId` hook that can be used to generate IDs for form elements etc. if needed.